external/common/templates/_secrets.tpl

```diff
@@ -114,12 +123,19 @@ The order in which this function returns a secret password:
     {{- $subStr := list (lower (randAlpha 1)) (randNumeric 1) (upper (randAlpha 1)) | join "_" }}
     {{- $password = randAscii $passwordLength }}
     {{- $password = regexReplaceAllLiteral "\\W" $password "@" | substr 5 $passwordLength }}
-    {{- $password = printf "%s%s" $subStr $password | toString | shuffle | b64enc | quote }}
```

Comment:
There are quite a few of these sprinkled around. It's definitely out of my comfort zone, but these are not customizations that were previously added by us?

Reply:
No I don't think we have customized these values. The external folder is meant to be a mirror.